### PR TITLE
[cmake] Fix windows build after migration to msys2

### PR DESCRIPTION
**Pull request by fetzerch**
This is a quick fix for the CMake Windows build after the migration msys2. I'm not fully happy with it as it's still required to run `make-mingwlibs.bat` for building those libs. I've put it to my todo list to build ffmpeg and libdvd on Windows directly from CMake.

### CODE REVIEW

File: project/cmake/CMakeLists.txt [hunk 1]
```diff
@@ -24,7 +24,6 @@ option(ENABLE_PLIST       "Enable AirPlay support?" ON)
 option(ENABLE_NFS         "Enable NFS support?" ON)
 option(ENABLE_AIRTUNES    "Enable AirTunes support?" ON)
 option(ENABLE_CEC         "Enable CEC support?" ON)
-option(ENABLE_OPENSSL     "Enable OpenSSL?" ON)
```
Comment (fetzerch) [inline project/cmake/CMakeLists.txt]: @wsnipex: openssl is not used on Windows, so we keep the option in line 41, right?

Comment (wsnipex) [inline project/cmake/CMakeLists.txt]: yep, fine

Comment (Paxxi) [inline project/cmake/CMakeLists.txt]: to be picky, we do use openssl on windows for curl and will be used for libssh as well, it's not needed for kodi build but it is needed for distribution. No clue if it's relevant, I just wanted to clarify.

Comment (fetzerch) [inline project/cmake/CMakeLists.txt]: Thanks. This option tells CMake to find the OpenSSL header and lib and link it to the Kodi executable. Curl and other dependencies on Windows are still being downloaded in binary form, so if that has OpenSSL all should be good.

**Pull request merged**